app/controllers/api/podcasts_controller.rb

```diff
@@ -13,7 +13,13 @@ def included(relation)
   end
 
   def show
-    super if visible?
+    if visible?
+      if request.format.rss?
+        render plain: FeedBuilder.new(show_resource).to_feed_xml
```

Comment:
For debugging.  Note that this only responds when you add a `/api/v1/podcasts/123.rss` extension (something about hal_api-rails doesn't let you request RSS here via `Accepts:`).  And this is cached.

The old (pre-Feeder-HTML) `/podcasts/123` endpoint worked a bit different.  But I couldn't quite get this to render that way, so ended up doing it manually.